package.json

```diff
@@ -67,8 +68,6 @@
     "url-loader": "^1.1.2",
     "webpack": "^4.29.6",
     "webpack-assets-manifest": "^3.1.1",
-    "webpack-cli": "^3.3.0",
-    "webpack-command": "^0.4.2",
```

Comment:
Wathchew got against webpack-command?

Reply:
It wouldn't run with webpack command. Let me try to fix!